src/pages/references/language-clarity-functions.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
@friedger could you update the title & descriptions as well?

Reply:
done